README.md

```diff
@@ -0,0 +1,70 @@
+# Nottify-React-Native
+
+---
+
+* This is a simple music app for android using React-Native
+
+# Features
+
+---
+
+* Search online songs (Require Network Permission)
```

Comment:
- User authentication
- Load songs from the device
- Play / Pause / Stop / Next / Previous / Shuffle / Loop / Time navigate
- Remember the last played song from the previous run
- Playlist: Add / Remove / Play / Pause songs from playlist
- Search and play online songs
- Background playback